tools/ci/azure/install_certs.yml

```diff
@@ -1,7 +1,8 @@
 steps:
 - script: |
     # https://github.com/web-platform-tests/results-collection/blob/master/src/scripts/trust-root-ca.sh
-    sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain tools/certs/cacert.pem
+    # only run this on macOS < 11
```

Comment:
Couldn't we just remove this whole step?

Reply:
we're still running Safari Stable on macOS 10.15, and we don't have `acceptInsecureCerts` support in stable